app/src/main/java/com/peacecorps/pcsa/LoginActivity.java

```diff
@@ -0,0 +1,81 @@
+package com.peacecorps.pcsa;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Spinner;
+import android.widget.Toast;
+
+public class LoginActivity extends AppCompatActivity  implements AdapterView.OnItemSelectedListener {
+
+    private Toolbar toolbar;
+    private Button loginButton;
+    private EditText name;
+    private Spinner country;
+    private String selected_country;
+    SharedPreferences.Editor editor;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+        toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+
+        editor = PreferenceManager.getDefaultSharedPreferences(LoginActivity.this).edit();
```

Comment:
Did you add a separate shared preference for this? Or is it the same as the one which is used for the tutorials?